packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -69,25 +69,25 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
     let overflowMenuId = this._id + '-overflow';
 
     return (
-      <div className={ css('ms-Breadcrumb', className) } ref='renderingArea'>
+      <div className={ css('ms-Breadcrumb', className, styles.root) } ref='renderingArea'>
```

Comment:
Seems intentional but just out of curiosity, are we still going to keep ms-Breadcrumb-* classes around when they are gone from scss?